### ISSUE TITLE
ScummVM Virtual Keyboard

### ISSUE THREAD

**Issue by killer101** [libretro-mirrors/scummvm#69]
I'm using the ScummVM engine in Lakka. As far as I can see, there is no option to bring up the virtual keyboard. As I found out, it has to be enabled while compiling the source code.

http://wiki.scummvm.org/index.php/Virtual_Keyboard

As some games are unplayable without simulating a keypress, I think this option is a must have!

**Comment by eku** [comment]
Even though this issue is very old, the problem still seems to exist. With a native installation of ScummVM there is a tab "Input" in the settings. This tab does not exist in the libretro version. Even the configuration of a hotkey in RetroArch to switch on the virtual keyboard does nothing in ScummVM libretro core.

**Comment by Darknior** [comment]
Yes it's a big problem for some games like Humongous one that use S and L to Save and Load.
We can't do it ... and when i try to do it with my real keyboard it not working too :(
We miss the virtual keyboard...
